Example_Card_Calypso/src/main/java/org/eclipse/keyple/card/calypso/example/UseCase12_PerformanceMeasurement_Validation/Main_PerformanceMeasurement_Validation_Pcsc.java

```diff
@@ -0,0 +1,286 @@
+/* **************************************************************************************
+ * Copyright (c) 2022 Calypso Networks Association https://calypsonet.org/
+ *
+ * See the NOTICE file(s) distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License 2.0 which is available at http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ************************************************************************************** */
+package org.eclipse.keyple.card.calypso.example.UseCase12_PerformanceMeasurement_Validation;
```

Comment:
I would rename the package org.eclipse.keyple.card.calypso.example.UseCase12_PerformanceMeasurement_**Embedded**Validation

Next for the reload measure, we will call it:
org.eclipse.keyple.card.calypso.example.UseCaseXX_PerformanceMeasurement_**Distributed**Reload